packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -570,5 +570,5 @@ For more info please visit [widget_toolkit_pin](https://pub.dev/packages/widget_
 [patrol_pub_lnk]: https://pub.dev/packages/patrol
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
-[rx_bloc_cli_cd_setup_lnk]: docs/continuous_delivery.md
-[golden_tests_with_smart_widgets_lnk]: docs/golden_tests.md
+[rx_bloc_cli_cd_setup_lnk]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/continuous_delivery.md
```

Comment:
I believe there is a misunderstanding

- The **brick** README.md file needs to use the relative path so that the links lead to the generated project
- The **example** README.md file needs to use the full URL  so that the links lead to the GitHub repository

Reply:
Well, even if we update the `example` only with full paths it will be overridden the next time the project is generated from the brick. However, I can write a script that will search for those cases in the example and replace them with the full URLs, if that's what we want.

Reply:
In long term it will be great to have this automation. For the purpose of this release we can do it manually

Reply:
Sure, I'll address it in a separate PR.